bot/blenderbot.py

```
@@ -1,8 +1,10 @@
 import requests,os
+
 try:
 	token = open(os.path.join('bot','token.txt')).read()
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)